arkane/encorr/corr.py

```diff
@@ -121,10 +122,12 @@ def get_atom_correction(model_chemistry, atoms, atom_energies=None):
             )
 
     # Step 2: Atom energy corrections to reach gas-phase reference state
-    atom_enthalpy_corrections = {symbol: data.atom_hf[symbol] - data.atom_thermal[symbol] for symbol in data.atom_hf}
+    atom_enthalpy_corrections = {symbol: (
+        data.atom_hf[symbol], data.atom_thermal[symbol]) for symbol in data.atom_hf}
```

Comment:
LGTM, @cgrambow, could you take a look at this line and approve as well?

Reply:
This introduces a large change of 17 kJ/mol for H2O2, from the tests:
```
FAIL: Test specifying absolute file paths of statmech files
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/arkane/statmechTest.py", line 193, in test_specifying_absolute_file_paths
    self.assertAlmostEqual(h2o2.conformer.E0.value_si, -146031.49933673252)
AssertionError: -128840.68059141742 != -146031.49933673252 within 7 places (17190.818745315104 difference)
```

If this was indeed a bug, then it's a big one. Let's be as confident as we can that this fix is the desired behaviour, i.e., we need a test set with trusted H298's to run on master and on this branch.

Reply:
The ATcT Hf0K for H2O2 is -129 kJ/mol (https://atct.anl.gov/Thermochemical%20Data/version%201.122g/species/?species_number=53) 
We get closer to that with this PR, if `conformer.E0` is suppose to be Hf0K